src/client/src/api/meals.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+const rootHost = 'http://localhost:5000/api/meals';
+
+const getAll = async () => {
+  const meals = await axios.get(`${rootHost}/`);
+  
+  return meals.data
+};
+
+const getById = async (id) => {
+  const meals = await axios.get(`${rootHost}/${id}`);
+  
+  return meals.data
+};
```

Comment:
Nice and clean! 
However, what if there is an error? 💡

Reply:
i decided to just use try catch in useEffect:
(`try {
        const res = await mealsAPI.getAll();
        setMeals(res);
      } catch (error) {
        return navigate('/');
      }`)
      
still working on it i think i will change it later